.github/workflows/build.yml

```diff
@@ -54,6 +54,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           path: src/flutter
+          fetch-depth: 0
```

Comment:
`fetch-depth: 0` means `fetch all history`, do we need all of this?

Reply:
The information is used by `format.dart` when finding the merge-base (`git merge-base FETCH_HEAD HEAD`).

Otherwise you will see this error: https://github.com/swift-kim/engine/runs/5227314578?check_suite_focus=true

Reply:
https://github.com/flutter-tizen/plugins/blob/03e18e3d5363fc46fac148823534e69fa558383c/.github/workflows/build.yml#L11

Maybe a depth of 2 would be enough?

Reply:
2 should not be enough for general cases. Fetching all branches and tags takes up to only 30 seconds so there should be no problem.

Reply:
Thanks for the detailed explanation!